templates/READMEs/README.md

```diff
@@ -1,107 +1,133 @@
-<!-- TODO: This is a template SDK README.md. It might also be useful for other repository types. Update the TODOs with correct examples/documentation. Replace all instances of "your-repo" with the appropriate name -->
+<!-- TODO: This is a template SDK README.md.
+It's structured to fit nicely into openfeature.dev, which is configured to include SDK READMEs.
+Complete all the TODOs applicable for your implementation and then create an issue in https://github.com/open-feature/openfeature.dev
+-->
 
 <!-- markdownlint-disable MD033 -->
+<!-- x-hide-in-docs-start -->
 <p align="center">
   <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/white/openfeature-horizontal-white.svg">
-    <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/black/openfeature-horizontal-black.svg">
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/white/openfeature-horizontal-white.svg" />
+    <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/black/openfeature-horizontal-black.svg" />
     <img align="center" alt="OpenFeature Logo">
   </picture>
 </p>
 
-<!-- TODO: update title -->
-<h2 align="center">OpenFeature your-repo</h2>
-
-<!-- TODO: add relevant badges -->
-[![Specification](https://img.shields.io/static/v1?label=Specification&message=v0.6.0&color=yellow)](https://github.com/open-feature/spec/tree/v0.6.0)
+<h2 align="center">OpenFeature <!-- TODO: your language SDK --> SDK</h2>
+
+<!-- x-hide-in-docs-end -->
+<!-- The 'github-badges' class is used in the docs -->
+<p align="center" class="github-badges">
+  <a href="https://github.com/open-feature/spec/tree/v0.6.0">
+    <img alt="Specification" src="https://img.shields.io/static/v1?label=specification&message=v0.6.0&color=yellow&style=for-the-badge" />
```

Comment:
We probably need to call out spec versions here so an implementer makes sure to update the values.

Reply:
I'll add a TODO, but keep it at 0.6.0 for now.